package.json

```diff
@@ -167,16 +170,18 @@
 		"sprintf-js": "1.1.1",
 		"style-loader": "1.0.0",
 		"stylelint-config-wordpress": "13.1.0",
-		"typescript": "3.5.3",
+		"typescript": "3.8.3",
 		"uuid": "3.3.2",
 		"webpack": "4.42.0",
 		"worker-farm": "1.7.0"
 	},
 	"scripts": {
 		"prebuild": "npm run check-engines",
-		"clean:packages": "rimraf ./packages/*/build ./packages/*/build-module ./packages/*/build-style ./packages/*/node_modules",
+		"clean:packages": "rimraf \"./packages/*/@(build|build-module|build-style)\"",
+		"clean:package-types": "tsc --build --clean",
```

Comment:
Shouldn’t it be part of clean:packages that is executed as prebuild:packages?

Reply:
There's some conversation around this in the thread here: https://github.com/WordPress/gutenberg/pull/18942#discussion_r396317889

In summary, there's enormous benefit to having a warm TypeScript cache and I don't believe there's any reason to clean it except to remove outdated declaration files from packages before they will be published.

In particular, we'll want to keep the caches warm for typechecking as part of lint staged.